src/EnergyPlus/HVACControllers.cc

```diff
@@ -375,6 +377,14 @@ namespace HVACControllers {
 				CheckEquipName( ControlNum ) = false;
 			}
 		}
+
```

Comment:
ManageContollers wil now return if controller is in OA system and instead be simulated in SolveWaterCoilController where controller will be simulated inside the iteration loop of OA sys water coils.